app/main/createWindow.js

```diff
@@ -41,6 +52,9 @@ export default ({ src, isDev }) => {
   // Setup event listeners for main window
   globalShortcut.register(shortcut, toggleMainWindow)
 
+  // Set the theme vibrancy from config
+  mainWindow.setVibrancy(themeMode(config.get('theme')))
```

Comment:
I mean it's fine if windows is just ignoring that, as long as there is no error or so :)